pkg/monitoring/hyperconverged/rules/alerts/operator_alerts.go

```diff
@@ -79,5 +81,17 @@ func operatorAlerts() []promv1.Rule {
 				healthImpactAlertLabelKey: "critical",
 			},
 		},
+		{
+			Alert: unsupportedArchitecturesAlert,
+			Expr:  intstr.FromString("kubevirt_hco_dict_with_no_supported_architectures == 1"),
+			Annotations: map[string]string{
+				"description": "the {{ $labels.data_import_cron_name }} DataImportCronTemplate (for the {{ $labels.data_source_name }} DataSource) has no supported architectures, and cannot be used to deploy the corresponding DataImportCron.",
+				"summary":     "DataImportCronTemplates with no supported architecture were detected in the HyperConverged resource.",
+			},
+			Labels: map[string]string{
+				severityAlertLabelKey:     "info",
+				healthImpactAlertLabelKey: "warning",
```

Comment:
@nunnatsa This label means the operator health is degraded.
1. Who is responsible for creating the DICT? Is this only Admins? Can regular users create this and potentially cause the operator to become degraded? I see Severity is info, so why health is warning.

2. If the DICT was just created, should it fire right away or wait?